lib/lambda-cfn.js

```diff
@@ -1002,36 +977,29 @@ function outputs(options) {
   return output;
 }
 
-function streambotEnv(options) {
-  if (!options.name)
-    throw new Error('name property required for streambotEnv');
+function envVariableParser(options) {
 
-    var env = {
-      "Type": "Custom::StreambotEnv",
-      "Properties": {
-        "ServiceToken": {
-          "Ref": "StreambotEnv"
-        },
-        "FunctionName": {
-          "Ref": options.name
-        }
-      }
-    };
+    var p = !options.parameters ? {} :
+        JSON.parse(JSON.stringify(parameters(options)));
 
-  var p = !options.parameters ? {} :
-      JSON.parse(JSON.stringify(parameters(options)));
+    for (var k in p) {
```

Comment:
Although it moved it around, i didnt change any of this compared to master:
https://github.com/mapbox/lambda-cfn/blob/master/lib/lambda-cfn.js#L1021

I can rename though to clear up the naming